docs/UserGuide.adoc

```diff
@@ -486,63 +486,138 @@ You can then run `sort rate` to obtain the list of donors with higher success ra
+
+* The task list can be updated using another command,`processingMarkDone ic/**PATIENT NRIC** ic/**DONOR NRIC** **TASK NUMBER**`
 
-After typing this command, the following changes occur:
+* The task list will be saved automatically.
 
-* A list will be generated to show the necessary tasks the hospital need to do for the respective donor and patient.
-* The list will be unique to the specific donor and patient pair
-* The list can be updated using `done** TASK NUMBER**` +
+* A donor can only be in `Processing` state with only one patient and vice versa.
+This means, if a donor and a patient is already in `Processing` state, they will not be able to be processed with another donor or patient.
 
-The list will be saved after the command `exit` is executed. The page will then stop displaying the list.
+* A screenshot of the list generated can be shown below:
+
+.Screenshot of the screen after you use the command on a valid donor and patient
+
+image::processing.png[width="790"]
 
 Format: `processing ic/**PATIENT NRIC** ic/**DONOR NRIC**` +
-The order of the NRICs does not matter.
+The order of the NRICs does not matter, as long as the patient and the donor must be valid.
 
 Example:
-`processing ic/S6764235G ic/S1234567D`
+`processing ic/S6764235G ic/S5234567D`
+
 // end::process[]
 
 // tag::mark[]
+
+=== Checking the task list : `processingMarkDone`
+
+As introduced above, this command serves to help you mark a task on the task list as done.
+This will allow you to be able to know what other administrative things you need to do
+for the specific donor and patient before the cross-matching test.
+Beside each task on the task list, there will be either a cross, ✘, or a tick, ✓.
+A tick would mean that the task is completed and a cross will mean that the task is not completed.
+
+.Screenshot of the screen after you use the command on a valid donor and patient and a valid task number
+
+image::processingMarkDone.png[width="790"]
+
+Format: `processingMarkDone ic/**PATIENT NRIC** ic/**DONOR NRIC** **TASK NUMBER**` +
+The order of the NRICs does not matter, as long as the task number, patient and donor must be valid.
+The donor and patient need to be in `Processing` state to be considered valid.
+
+Example:
+`processingMarkDone ic/S6764235G ic/S5234567D 1`
+
+// end::process[]
+
+// tag::mark[]
+
 === Mark as done : `done`
 
 After the hospital has the results of the cross matching test, they can
-update the results in ORGANice. Cross matching tests have two possible results: pass or fail.
+update the results in ORGANice.
+Cross matching tests have two possible results: pass or fail.
 
-Format: `done icP/**NRIC OF PATIENT** icD/**NRIC OF DONOR** res/[pass/fail]`
-
-`icP/**NRIC OF PATIENT**` refers to the NRIC of the patient, while `icD/**NRIC OF DONOR**`
-refers to the NRIC of the donor.
+Format: `done ic/**PATIENT NRIC** ic/**DONOR NRIC** res/[pass/fail]`
+The order of the NRICs does not matter, as long as the patient and the donor must be valid.
```

Comment:
suggested addition: must **both** be valid

Reply:
amended